core/dbt/cli/params.py

```diff
@@ -358,9 +358,20 @@
 )
 
 warn_error = click.option(
-    "--warn-error/--no-warn-error",
+    "--warn-error",
     envvar="DBT_WARN_ERROR",
     help="If dbt would normally warn, instead raise an exception. Examples include --models that selects nothing, deprecations, configurations with no associated models, invalid test configurations, and missing sources/refs in tests.",
+    default=None,
+    flag_value=True,
+)
+
+warn_error_options = click.option(
+    "--warn-error-options",
+    envvar="DBT_WARN_ERROR_OPTIONS",
+    default=None,
+    help="""If dbt would normally warn, instead raise an exception based on include/exclude configuration. Examples include --models that selects nothing, deprecations, configurations with no associated models, invalid test configurations,
```

Comment:
careful, this is how you prove you've been using dbt for a while :)

`--select` has been canonical since v0.21
```suggestion
    help="""If dbt would normally warn, instead raise an exception based on include/exclude configuration. Examples include --select that selects nothing, deprecations, configurations with no associated models, invalid test configurations,
```